File: ci/src/freecam_ci/project_files.py
from pathlib import Path

CI = Path(__file__).resolve().parents[2]
ROOT = CI.parent
CHANGELOG_FILE = ROOT / "CHANGELOG.md"
METADATA_FILE = ROOT / "metadata.toml"
STONECUTTER_FILE = ROOT / "stonecutter.json5"
MATRIX_JOBS_FILE = CI / "matrix_jobs.toml"

File: ci/src/project_files.ts
import path from "node:path";

export const CI = path.resolve(import.meta.dirname, "..");
export const ROOT = path.resolve(CI, "..");
export const CHANGELOG_FILE = path.resolve(ROOT, "CHANGELOG.md");
export const METADATA_FILE = path.resolve(ROOT, "metadata.toml");
export const STONECUTTER_FILE = path.resolve(ROOT, "stonecutter.settings.toml");
export const MATRIX_JOBS_FILE = path.resolve(CI, "matrix_jobs.toml");
